app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+    belongs_to :post
```

Comment:
- [OPTIONAL] The `Comment` model should belong to the `User` also but you didn't define it in this model class. I would advise you to define it so that other instances of the `Comment` model can be created properly. 💻 🖱️ 

```ruby
 belongs_to :author, class_name: 'User'
```